src/lib/components/schedule/schedules-table-row.svelte

```diff
@@ -87,6 +83,17 @@
     {/each}
   </td>
 </TableRow>
+<TableRow>
```

Comment:
Do users need to see the entire `Schedule Spec` in this view? Or would even something smaller like this be adequate? 

<img width="1285" alt="Screenshot 2023-08-30 at 11 58 01 AM" src="https://github.com/temporalio/ui/assets/15069288/86903bd6-dec4-4aa0-941a-8e173468d472">

Or maybe we could add a `!border-t-0` to remove the top border so the row is more visually connected? 

<img width="1289" alt="Screenshot 2023-08-30 at 12 01 51 PM" src="https://github.com/temporalio/ui/assets/15069288/e6101a3a-a36f-4393-989d-b29e16e12f2c">